spark_parser/ast.py

```diff
@@ -28,7 +30,11 @@ def __hash__(self):
         return hash(self.type)
 
     def __repr__(self, indent=''):
+        global current_indsym #STRANIKS_SCAN 04/04/2016
```

Comment:
Instead of a global variable, why not use an instance variable instead? That way, a program using this can have several parsers running in different threads in parallel. 

For this reason, using global variables for changing state is not a good idea.